muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptofRecordsJsPlugin_en.properties

```diff
@@ -41,7 +41,10 @@ plugin.records.summary.notifications.title = Notifications
 plugin.records.yo.title = Your matricular examinaition
 plugin.records.yo.abiStatus.title = ABI-status
 plugin.records.yo.abiStatus.content.finished = Your ABI-status is does not lack courses. You can now apply for matricular examination.
```

Comment:
Not part of this PR but should be fixed nevertheless. Suggested change: *You have achieved ABI-status and can now apply for matriculation examination.*